### ISSUE TITLE
Scroll to search result after navigation

### ISSUE THREAD

**Issue by ricardoboss** [trenz-gmbh/trenz-docs#17]
Currently, when clicking on a search result, it just takes the user to the page itself, not the part where the search term is located.

As I see it currently, we have two options to improve the users experience:

- Find the nearest parent heading and modify the anchor of the search result URL to jump to this heading
- Use the new 'scroll-to-text-fragment' feature some browsers already implement ([stackoverflow question](https://stackoverflow.com/questions/62161819/what-exactly-is-the-text-location-hash-in-an-url), [caniuse](https://caniuse.com/url-scroll-to-text-fragment), [docs](https://web.dev/text-fragments/), [spec](https://wicg.github.io/scroll-to-text-fragment/))

The second option is only supported in chromium-based browsers.

**Comment by chucker** [comment]
Seems like it'll be a while until WebKit supports it, so we should probably go with the first for now?

**Comment by ricardoboss** [comment]
What about a feature switch based on the detected browser? Implement both and choose at runtime?

**Comment by chucker** [comment]
We can do that, but you'd have to hardcode user agents. (FWIW, Mozilla doesn't seem to be sure yet if a competing spec is better: https://mozilla.github.io/standards-positions/#fragmention)

**Comment by ricardoboss** [comment]
Yeah ok, might better to stick to the standards on this one. So nearest parent heading it is?

**Comment by chucker** [comment]
Yeah, IMHO, just go with nearest for now and reëvaluate in a year or so.
